composer.json

```diff
@@ -39,25 +39,25 @@
         "sonata-project/doctrine-extensions": "^1.10.1",
         "sonata-project/form-extensions": "^0.1.2 || ^1.4",
         "sonata-project/twig-extensions": "^0.1.1 || ^1.3",
-        "symfony/config": "^4.4",
-        "symfony/console": "^4.4",
-        "symfony/dependency-injection": "^4.4",
-        "symfony/event-dispatcher": "^4.4",
+        "symfony/config": "^4.4 || ^5.1",
```

Comment:
these composer changes should be done on the current stable branch

Reply:
I think they are done on master because it require BC-break, no ?

It's like SonataAdmin which only support SF 5 on master branch

Reply:
@VincentLanglet exactly thats what i thought as well